spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/HyperConformance.scala

```diff
@@ -145,8 +149,17 @@ object HyperConformance extends StreamTransformerFactory with HyperConformanceAt
     implicit val reportDateCol: InfoDateFactory = InfoDateFactory.getFactoryFromConfig(conf)
     implicit val infoVersionCol: InfoVersionFactory = InfoVersionFactory.getFactoryFromConfig(conf)
 
-    implicit val menasBaseUrls: List[String] = MenasConnectionStringParser.parse(conf.getString(menasUriKey))
-    new HyperConformance()
+    val menasBaseUrls: List[String] = MenasConnectionStringParser.parse(conf.getString(menasUriKey))
+    val menasBaseUrlsTryCounts: List[Int] = if (conf.containsKey(menasUriTriesKey)) {
+      val value = conf.getString(menasUriTriesKey)
+      val configReader = new ConfigReader(ConfigFactory.empty()
```

Comment:
Using `ConfigReader` seemed as the easiest solution to ensure read compatibility of the configurations.